migration/iD12/postgresql/202404301200_IDEMPERIE-5329.sql

```diff
@@ -0,0 +1,289 @@
+;
+
+-- Adding Currency field on Deposit batch window
+-- Apr 4, 2024, 6:01:37 PM IST
+UPDATE AD_Column SET AD_Val_Rule_ID=NULL, ColumnName='C_BankAccount_ID', AD_Reference_Value_ID=NULL, Callout='org.compiere.model.CalloutDepositBatch.bankAccount', AD_Process_ID=NULL, IsSyncDatabase='Y', AD_Chart_ID=NULL, PA_DashboardContent_ID=NULL, AD_Val_Rule_Lookup_ID=NULL,Updated=TO_TIMESTAMP('2024-04-04 18:01:37','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=208412
+;
+
+-- Apr 4, 2024, 6:01:38 PM IST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintName,FKConstraintType,IsHtml) VALUES (216588,0,'Currency','The Currency for this record','Indicates the Currency to be used when processing or reporting on this record',200056,'C_Currency_ID',22,'N','N','N','N','N',0,'N',19,0,0,'Y',TO_TIMESTAMP('2024-04-04 18:01:37','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-04-04 18:01:37','YYYY-MM-DD HH24:MI:SS'),100,193,'Y','N','D','Y','N','N','Y','0cec46ff-7504-42bf-a823-90399ce75d81','Y',0,'N','N','CCurrency_CDepositBatch','N','N')
+;
+
+-- Apr 4, 2024, 6:01:38 PM IST
+ALTER TABLE C_DepositBatch ADD COLUMN C_Currency_ID NUMERIC(10) DEFAULT NULL 
```

Comment:
the changes to deposit batch window is not documented in the wiki doc so I'm not sure whether it is necessary to add the currency field here. Looking at the data model, c_bankaccount_id is not null and you can always get the c_currency_id value from c_bankaccount table.